[DOCU-2402] Remove KM and decK, and HTTPie instructions from comprehensive docs. (#4137)
Kong/docs.konghq.com: src/gateway/get-started/comprehensive/load-balancing.md

In this section, you will create an Upstream named `example_upstream` and add two targets to it.

Call the Admin API on port `8001` and create an Upstream named `example_upstream`:

```sh

> curl -X POST http://<admin-hostname>:8001/upstreams \

Reviewer: ```suggestion
curl -X POST http://localhost:8001/upstreams \
```

  --data name=example_upstream
```
Update the service you created previously to point to this upstream:

```sh

> curl -X PATCH http://<admin-hostname>:8001/services/example_service \

Reviewer: ```suggestion
curl -X PATCH http://localhost:8001/services/example_service \
```

  --data host='example_upstream'
```

Add two targets to the upstream, each with port 80: `mockbin.org:80` and
`httpbin.org:80`:

```sh

> curl -X POST http://<admin-hostname>:8001/upstreams/example_upstream/targets \

Reviewer: ```suggestion
curl -X POST http://localhost:8001/upstreams/example_upstream/targets \
```

  --data target='mockbin.org:80'

> curl -X POST http://<admin-hostname>:8001/upstreams/example_upstream/targets \

Reviewer: ```suggestion
curl -X POST http://localhost:8001/upstreams/example_upstream/targets \
```

  --data target='httpbin.org:80'
```

You now have an Upstream with two targets, `httpbin.org` and `mockbin.org`, and a service pointing to that Upstream.

## Validate the Upstream Services

> 1. With the Upstream configured, validate that it’s working by visiting the route `http://<admin-hostname>:8000/mock` using a web browser or CLI.

Reviewer: ```suggestion
1. With the Upstream configured, validate that it’s working by visiting the route `http://<localhost:8000/mock` using a web browser or CLI.
```
